depfinder/utils.py

```diff
@@ -68,10 +69,10 @@
     Loader=yaml_loader,
 )
 
-req = requests.get('https://raw.githubusercontent.com/regro/cf-graph-countyfair/master/mappings/pypi/name_mapping.yaml')
-if req.status_code == 200:
-    mapping_list = yaml.load(req.text, Loader=yaml_loader)
-else:
+try:
+    import conda_forge_metadata.autotick_bot
+    mapping_list = conda_forge_metadata.autotick_bot.get_pypi_name_mapping()
+except (ImportError, AttributeError, requests.exceptions.HTTPError):
     mapping_list = yaml.load(
```

Comment:
log the exception here too please. something like `logger.exception('error encountered. conda-forge metadata not available')`